.travis.yml

```diff
@@ -8,6 +8,8 @@ addons:
     - ubuntu-toolchain-r-test
     packages:
     - g++-4.8
+services:
+  - redis-server
```

Comment:
We need another redis instance to test the default connection and I thought that's an easy way to handle this.